test/app-luatest/system_fiber_test.lua

```diff
@@ -0,0 +1,45 @@
+local server = require('test.luatest_helpers.server')
+local g = require('luatest').group()
+
+g.before_all = function()
+    g.server = server:new{alias = 'default'}
+    g.server:start()
+    g.server:exec(function(uri)
+        rawset(_G, 'fiber', require('fiber'))
+        rawset(_G, 'conn', require('net.box').connect(uri))
+        rawset(_G, 'is_system', function(name)
+            return name:endswith('(net.box)') or
+                   name == 'gc' or name == 'checkpoint_daemon' or
+                   name:startswith('vinyl.')
+        end)
+    end, {g.server.net_box_uri})
+end
+
+g.after_all = function()
+    g.server:drop()
+end
+
+g.test_system_fibers = function()
+    g.server:exec(function()
+        -- If the fiber is killed (which is not what we want, btw),
+        -- it is not shown in fiber.info(). So, in order to check
+        -- if the system fiber is still alive after kill we need to
+        -- save them in global variable.
+        rawset(_G, 'system_fibers', {})
+        for fid, f in pairs(_G.fiber.info()) do
+            if _G.is_system(f.name) then
+                -- fiber.find() + fiber:cancel() == fiber.kill()
+                local fiber_temp = _G.fiber.find(fid)
+                table.insert(_G.system_fibers, fiber_temp)
+                fiber_temp:cancel()
+            end
+        end
+    end)
+
+    g.server:exec(function()
```

Comment:
Why do you need the second exec? Can't you put all in one exec, without using `_G`?

Reply:
You're right. There's no need for `_G` at all. The test looks much cleaner without it.